### ISSUE TITLE
[Bug]: NeXTSTEP plist format doesn't work with FINALPACKAGE=1

### ISSUE THREAD

**Issue by asdfzxcvbn** [theos/theos#779]
### What are the steps to reproduce this issue?

1. make new project with example plist: `{ Filter = { Bundles = ( "hi" ); }; }`
2. `make clean package FINALPACKAGE=1`
3. fail at making stage.

### What happens?

everything compiles fine without `FINALPACKAGE=1`. however, compiling will fail at the making stage if `FINALPACKAGE=1` is set.

### What were you expecting to happen?

a non-debug deb is built.

### Paste any relevant logs, error output, etc.

```shell
$ make clean package FINALPACKAGE=1
==> Cleaning…
> Making all for tweak hey…
==> Preprocessing Tweak.xm…
==> Compiling Tweak.xm (arm64)…
==> Linking tweak hey (arm64)…
==> Generating debug symbols for hey…
==> Stripping hey (arm64)…
==> Merging tweak hey…
==> Signing hey…
> Making stage for tweak hey…
Loading '/tmp/hey/.theos/_/Library/MobileSubstrate/DynamicLibraries/hey.plist' - non-NSData data argument passed to method
make: *** [/home/zx/theos/makefiles/package.mk:11: internal-package] Error 1
```

### What OS are you seeing the problem on?

Linux/WSL

### What OS version does it have installed?

arch linux; kernel v6.6.1-arch1-1

### What toolchain and version are you using?

clang v16.0.6

### Which SDK version are you using?

iOS 16.5

### What OS is your client device running?

iOS

### What OS version does it have installed?

iOS 16.7

### Which device model is it?

iPhone 8

**Comment by L1ghtmann** [comment]
This seems to spawn from your plist converter. Can you run
```sh
if command -v plutil &> /dev/null; then
    echo $(which plutil)
elif command -v ply &> /dev/null; then
    echo $(which ply)
elif command -v plistutil &> /dev/null; then
    echo $(which plistutil)
else
    echo "none"
fi
``` 
and share the output, please?

**Comment by asdfzxcvbn** [comment]
> This seems to spawn from your plist converter. Can you run
> 
> ```shell
> if command -v plutil &> /dev/null; then
>     echo $(which plutil)
> elif command -v ply &> /dev/null; then
>     echo $(which ply)
> elif command -v plistutil &> /dev/null; then
>     echo $(which plistutil)
> else
>     echo "none"
> fi
> ```
> 
> and share the output, please?

`/sbin/plutil`

**Comment by L1ghtmann** [comment]
I don't believe Linux has a functioning `plutil`, so you may want to try either [`plistutil`](https://github.com/libimobiledevice/libplist) or [`ply`](https://github.com/DHowett/go-plist/tree/main/cmd/ply). The former can be installed by cloning and running
```
./autogen.sh --without-cython
make -j$(nproc --all) install 
```
whereas the latter is installed via a `go` command.

**Comment by asdfzxcvbn** [comment]
> I don't believe Linux has a functioning `plutil`, so you may want to try either [`plistutil`](https://github.com/libimobiledevice/libplist) or [`ply`](https://github.com/DHowett/go-plist/tree/main/cmd/ply). The former can be installed by cloning and running
> 
> ```
> ./autogen.sh --without-cython
> make -j$(nproc --all) install 
> ```
> 
> whereas the latter is installed via a `go` command.

i already have plistutil installed and it works fine, i even manually use it myself sometimes

**Comment by L1ghtmann** [comment]
Okay. In order to fix the issue you're seeing with release builds, you'll need to prioritize that over the plutil you have. That can be done a number of ways, but I would recommend uninstall plutil if plistutil meets your needs.
